doc/1.manual/x1.htm

```diff
@@ -96,8 +96,7 @@ <H3> <A id=s2> 1.2. other resources </A> </H3>
 for people interested in graphics, there is a 3D graphics rendering package,
 named GEM, based on OpenGL, written by Mark Danks, adapted to Linux by
 Guenter Geiger, and now maintained by IOhannes zm&ouml;lnig.  You can get
-it from: <A href="http://gem.iem.at/">http://gem.iem.at/</A>, or by
-installing Pd Extended.
+it from: <A href="http://gem.iem.at/">http://gem.iem.at/</A>, or using the externals manager "Deken", see <A href="x4.htm#s2.2"> 4.2.2. How to Download Externals from Pd Vanilla </A>.
```

Comment:
i'm not overly fond of installing complex externals with lots of dependencies (like `Gem`) via `deken`.
apart from that, you should remove the leading/trailing spaces in the link.

Reply:
> you should remove the leading/trailing spaces in the link.

you mean the trailing spaces here : `> 4.2.2. How to Download Externals from Pd Vanilla </A>.` ?